### ISSUE TITLE
[Bug] High number of albums causes genre list to freeze

### ISSUE THREAD

**Issue by espector** [jmshrv/finamp#684]
There's a new bug in the latest TestFlight build. For genres with many albums the list just freezes and doesn't display anything. 

This wasn't an issue with previous builds. 

including images to compare Jellyfin vs Finamp list for a genre. 
![IMG_1440](https://github.com/jmshrv/finamp/assets/3356907/783d30dc-60fd-4d5f-b2e4-a773ef07065b)
![IMG_1439](https://github.com/jmshrv/finamp/assets/3356907/b47791b9-9148-4a9a-9917-8aa1adefdd29)
![IMG_1438](https://github.com/jmshrv/finamp/assets/3356907/59c26403-31f1-40db-bf83-4c9faedce978)
![IMG_1437](https://github.com/jmshrv/finamp/assets/3356907/0a9c11ba-9304-4bf2-9160-b687b008808a)

**Comment by Chaphasilor** [comment]
It doesn't load for you at all?  
It's a known issue that it might take a few seconds to load (which of course isn't ideal, but it's a beta), but I wasn't aware of it not loading at all...

**Comment by espector** [comment]
That's correct. It loads when there aren't too many albums. I think I noticed the pattern. When Jellyfin has to paginate. Perhaps if there's more than 100 albums for a genre. 

Attaching an example where it loads fine (without pagination). 

I was also able to confirm this behavior with a fresh install on macOS. 

![IMG_1495](https://github.com/jmshrv/finamp/assets/3356907/3c535c0d-6707-4392-bf8a-bba89d1ed59e)
![IMG_1494](https://github.com/jmshrv/finamp/assets/3356907/0a3f4bd0-c910-48e2-b750-802fdcfcb8e2)

**Comment by espector** [comment]
I switched back to the App Store (non-Test flight) version, and the albums are loading fine there. So it's definitely the latest Test flight version.

**Comment by Chaphasilor** [comment]
This has now been resolved as part of [beta version 0.9.18](https://github.com/jmshrv/finamp/releases/tag/0.9.18-beta)!
